packages/app/src/app/pages/common/Modals/MoveSandboxFolderModal/index.tsx

```diff
@@ -19,20 +19,23 @@ import { DirectoryPicker } from './DirectoryPicker';
 export const MoveSandboxFolderModal: FunctionComponent = () => {
   const {
     actions: { dashboard, refetchSandboxInfo, modals: modalsActions },
-    state: { activeTeamInfo, activeTeam, modals },
+    state: { activeTeam, modals, activeTeamInfo },
   } = useOvermind();
   const [error, setError] = useState(undefined);
   const [loading, setLoading] = useState(false);
   const [path, setPath] = useState<string | null>(
     modals.moveSandboxModal.defaultOpenedPath
   );
   const [teamId, setTeamId] = useState(activeTeam);
-
-  const [selectedTeam, setSelectedTeam] = useState({
-    id: activeTeamInfo.id,
-    name: activeTeamInfo.name,
-    avatarUrl: activeTeamInfo.avatarUrl,
-  });
+  const [selectedTeam, setSelectedTeam] = useState(
```

Comment:
This could break the modal I think. If this happens

1. Modal renders without `activeTeamInfo`
2. `activeTeamInfo` becomes available
3. Now it's not in the modal

Is that a flow that would break?

Reply:
Yeah I think then the modal will never render if team is not available.

Reply:
Probably the check needs to be one leve higher

```
{activeTeamInfo && <MoveSandboxModal />}
```

Reply:
https://github.com/codesandbox/codesandbox-client/pull/5292